core/primitives/src/version.rs

```diff
@@ -22,7 +22,7 @@ pub const DB_VERSION: DbVersion = 30;
 pub use near_primitives_core::types::ProtocolVersion;
 
 /// Oldest supported version by this client.
-pub const OLDEST_BACKWARD_COMPATIBLE_PROTOCOL_VERSION: ProtocolVersion = 34;
+pub const OLDEST_BACKWARD_COMPATIBLE_PROTOCOL_VERSION: ProtocolVersion = 39;
```

Comment:
let's make sure that other PR gets merged before this one.

Reply:
Ok, I made this into draft.